.vib/fluentd/goss/vars.yaml

```diff
@@ -0,0 +1,36 @@
+binaries:
+  - fluentd
+  - ruby
+directories:
+  - mode: "0775"
+    paths:
+      - /opt/bitnami/fluentd/gems
+      - /opt/bitnami/fluentd/specifications
+      - /opt/bitnami/fluentd/cache
+      - /opt/bitnami/fluentd/doc
+      - /opt/bitnami/fluentd/conf
+      - /opt/bitnami/fluentd/logs
+      - /opt/bitnami/fluentd/plugins
+modules:
+  gems:
+    - fluent-plugin-concat
+    - fluent-plugin-detect-exceptions
+    - fluent-plugin-elasticsearch
+    - fluent-plugin-grafana-loki
+    - fluent-plugin-kafka
+    - fluent-plugin-kubernetes_metadata_filter
+    - fluent-plugin-multi-format-parser
+    - fluent-plugin-prometheus
+    - fluent-plugin-rewrite-tag-filter
+    - fluent-plugin-s3
+    - fluent-plugin-systemd
+    - fluent-plugin-record-modifier
+    - oj
+    - json
+    - fluentd
+    - activesupport
+    - elasticsearch-xpack
```

Comment:
This is okay, but I would select a subset of the most relevant ones. Again, this is subjective, so feel free to disregard my comment if you believe all of them are relevant.

Reply:
I believe that all packages should be reviewed even if there are several to review.